docs/templating/twig_function_reference/catalog_twig_functions.md

```diff
@@ -0,0 +1,49 @@
+---
+description: Catalog Twig functions enable getting and rendering certain catalog information and listing filter templates.
+---
+
+# Catalog Twig functions
+
+With the catalog Twig functions you can get catalog location, render catalog status and list filter templates.
+
+### `ibexa_get_filter_preview_templates`
+
+The `ibexa_get_filter_preview_templates()` function gets a list of filter preview Twig templates 
+(configured in `ibexa.repositories.product_catalog.filter_preview_templates`). 
+Then it sorts the list by priority and renders the filter templates.
+
+``` html+twig
+        {% for template in ibexa_get_filter_preview_templates() %}
+            {% if block(block_name, template) is defined %}
+                {{ block(block_name, template) }}
+
+                {% set block_found = true %}
+            {% endif %}
+        {% endfor %}
+```
+
+### `ibexa_get_product_catalog_root`
+
+The `ibexa_get_product_catalog_root()` function gets a remote ID of a location of the product catalog (configured in `ibexa_product_catalog.engines.default.type.options.root_location_remote_id`).
```

Comment:
`ibexa_get_product_catalog_root` returns product catalog root location, not "remote ID of a location of the product catalog"

Reply:
That's correct, I might have suggested the wrong description to @dabrt.